waspc/data/Generator/templates/server/src/email/core/helpers.ts

```diff
@@ -0,0 +1,25 @@
+{{={= =}=}}
+
+import { EmailFromField } from "./types";
+
+export function createFromEmailString({
+  email,
+  title,
+}: {
+  email: string;
+  title?: string;
+}): string {
+  if (title) {
+    return `${title} <${email}>`;
```

Comment:
SMTP from field can look like `Name <email@bla.com>` and it will look nice in the e-mail client.

Reply:
Can we maybe rename `title` to `name`? I think that is a bit more common (but could be wrong).

Reply:
Makes sense, I've renamed it to `name` since a bit more common then `title`